introduction.tex

```diff
@@ -7,7 +7,8 @@ \chapter{Introduction}
 different criticality were isolated by hardware.  This approach is no longer practical as it has
 proven inefficient and restrictive.  The result is \emph{mixed-criticality} systems, where software
 applications with different criticalities execute on the same hardware.
-
+% Does a software application have mixed-criticality components or single criticality.
+% Application, system, functional componet ?
```

Comment:
I guess everyone has different ways of referring to functional components. Does a system have multiple applications and then each application has multiple components.  Can components only have one criticality level, but applications have multiple. Or do applications == components, and applications have 1 criticality level and systems have multiple applications with different criticality levels.